src/main/java/com/google/devtools/build/lib/bazel/repository/downloader/DownloadManager.java

```diff
@@ -101,8 +102,11 @@ public Path download(
     }
 
     List<URL> rewrittenUrls = originalUrls;
+    Map<URI, Map<String, String>> rewrittenAuthHeaders = ImmutableMap.copyOf(authHeaders);
```

Comment:
nit: `rewrittenAuthHeaders = authHeaders` to save the copy?

Reply:
Please address this comment or I can do it during import if you don't mind.

Reply:
@ma-oli I submitted https://github.com/ma-oli/bazel/pull/1 to do this in case that's helpful!

Reply:
I will import the PR and address this during that. Thanks for the PR!

Reply:
Thanks!

Reply:
Hey; sorry about that. I'm not super familiar with Github and I didn't receive these on the right email. Thanks a lot @keith for following up.